examples/placeholder/linux/include/TestCommand.h

```diff
@@ -129,7 +129,7 @@ class TestCommand : public TestRunner,
     {
         if (commandPath == mCommandPath)
         {
-            NextTest();
+            chip::DeviceLayer::PlatformMgr().ScheduleWork(ScheduleNextTest, reinterpret_cast<intptr_t>(this));
```

Comment:
This seems ok, but shouldn't the `RemoveEventHandler` bit move from ScheduleNextTest into the OnPlatformEvent case where we get the kCommissioningComplete event?

Also, should we be setting isRunning to false somewhere given that ScheduleNextTest sets it to true?  Not sure what the meaning of the member is...

Reply:
It is only set to false when waiting for commissioning, so setting this to True is a no-op since it should already be True in the other scenarios. I also noticed `RemoveEventHandler`, I thought it was a no-op if the event didn't exist so I didn't plan to move, but it makes more sense to move it.

Reply:
Ok, moved it.

Reply:
Re-tested and the issue remains resolved.